src/parser.ts

```diff
@@ -5,12 +5,14 @@
  * @copyright jQuery Foundation and other contributors, https://jquery.org/
  * MIT License
  */
+import calculateProjectParserOptions from './tsconfig-parser';
 import semver from 'semver';
 import ts from 'typescript';
 import convert from './ast-converter';
+import util from './node-utils';
 import { Extra, ParserOptions } from './temp-types-based-on-js-source';
 
-const packageJSON = require('../package.json');
+import packageJSON from '../package.json';
```

Comment:
See my other comment about this altering the `dist` structure, I believe that is why I `require`d it here

Reply:
Reverted back to `require`.